lms/models/assignment.py

```diff
@@ -74,6 +76,19 @@ class Assignment(CreatedUpdatedMixin, BASE):
     description = sa.Column(sa.Unicode, nullable=True)
     """The resource link description from LTI params."""
 
+    @property
+    def group_set_id(self) -> Optional[str]:
+        """Get the ID of the group set this assignment is configured with."""
+
+        if group_set_id := self.extra.get("group_set_id"):
+            # Different LMS's use integers/strings/uuids.
+            # Always return a string here an let the specific
+            # API services deal with the differences.
```

Comment:
See for example: 

https://github.com/hypothesis/lms/blob/main/lms/services/canvas_api/client.py